scripts/tag.sh

```diff
@@ -5,8 +5,6 @@ set -e
 echo "Enter Git Tag"
 read -r TAG
 
-git add .
-git commit -m "$TAG"
-git tag "$TAG"
-git push origin "$TAG"
-git push origin master
\ No newline at end of file
+echo "Bump iOS and Android"
+fastlane bump
```

Comment:
The TAG is useless here. What about moving `fastlane bump` in a npm script?

Reply:
the pkg.json version is used by fastlane bump, so must be incremented via yarn version, which also does git tag under the hood